components/automate-ui/src/app/components/sidebar/sidebar.component.ts

```diff
@@ -15,7 +15,7 @@ export class SidebarComponent {
   constructor(
     @Inject(LayoutFacadeService) layoutFacade: LayoutFacadeService
   ) {
-    this.menuGroups$ = layoutFacade.menuGroups$;
+    this.menuGroups$ = layoutFacade.sidebar$;
   }
 
   public isVisible(item: any) {
```

Comment:
Can stongly type the return value.
```suggestion
  public isVisible(item: any): Observable<boolean> {
```
I do not see any easy fix for the parameter, but this helps, I think:
```suggestion
  public isVisible(item: any /* boolean | Observable<boolean> */): Observable<boolean> {
```

That said, I do not see anywhere that you pass an `Observable` -- every occurrence is a plain boolean, so could simplify this function, I think, to just return a boolean
(and also change the MenuItemGroup.visible to be boolean instead of any in layout.model.ts).

Reply:
@msorens It was previously passing either a boolean or observable. I now have control over that and can remove the observable logic.👍🏻